static/js/util/sorting.js

```diff
@@ -1,10 +1,12 @@
 // @flow
```

Comment:
I think that ease of testing / separation from the react end of things is one reason to put functions like this in a separate file. w/r/t to sorting the work history entries that would only live on one component, but we use two different components to display the education stuff (`EducationDisplay` and `EducationForm`) so I'd need to have some way of sharing the function between those two components in any case.